connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -755,4 +755,14 @@ private static Object convertToConnect(Schema schema, JsonNode jsonValue) {
     private interface LogicalTypeConverter {
         Object convert(Schema schema, Object value);
     }
+    
+    //Exists for testing purposes
+    public boolean convertToJsonEqual(Schema schema, Struct value) {
```

Comment:
If this is only for testing then you should at least make it package-private scope or move it to separate util class (in test of course).